make.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function make(argument) {
+  let argList = [];
+  function f(arg) {
+    if (arg === undefined) throw new Error('Callback is undefined');
```

Comment:
` arg === undefined` the same `if (!arg)`
`new Error('Callback is undefined');` why  callback? arg not only callback